src/Sentry/SentryLogger.cs

```diff
@@ -0,0 +1,68 @@
+using Sentry.Experimental;
+using Sentry.Infrastructure;
+using Sentry.Protocol.Envelopes;
+
+//TODO: add XML docs
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace Sentry;
+
+/// <summary>
+/// Creates and sends logs to Sentry.
+/// </summary>
+[Experimental(DiagnosticId.ExperimentalSentryLogs, UrlFormat = UrlFormats.ExperimentalSentryLogs)]
+public sealed class SentryLogger
+{
+    //TODO: QUESTION: Trace vs LogTrace
+    // Trace() is from the Sentry Logs feature specs. LogTrace() would be more .NET idiomatic
+    public void Trace(string template, object[]? parameters = null, Action<SentryLog>? configureLog = null)
```

Comment:
Hm, could we have overloads with `T` instead to avoid boxing cost when logging is disabled?

That's the approach taken on the diagnostic logger extensions

Reply:
Yes, it's also on my bucket list ... I updated the description to reflect required work, too.
I'd not only like to add up to 3 `TArg` overloads per severity, but also - in a follow-up PR - add a custom `[InterpolatedStringHandler]`, which I'm going to explore over the weekend.